src/Operations.jl

```diff
@@ -586,12 +587,36 @@ function install_git(
     end
 end
 
+function collect_artifacts(pkg_root::String; platform::AbstractPlatform=HostPlatform())
+    # Check to see if this package has an (Julia)Artifacts.toml
+    artifacts_tomls = Tuple{String,Dict}[]
+    for f in artifact_names
+        artifacts_toml = joinpath(pkg_root, f)
+        if isfile(artifacts_toml)
+            selector_path = joinpath(pkg_root, ".pkg", "select_artifacts.jl")
+
+            # If there is a dynamic artifact selector, run that in an appropriate sandbox to select artifacts
+            if isfile(selector_path)
+                select_cmd = Cmd(`$(gen_build_code(selector_path)) --project=$(project) $(triplet(platform))`)
```

Comment:
This doesn't actually do anything with this load path, and code should not be using this env. `project` also seems undefined?
```suggestion
                select_cmd = Cmd(`$(gen_build_code(selector_path)) $(triplet(platform))`)
```